cluster/gce/util.sh

```diff
@@ -868,7 +890,13 @@ function construct-linux-kubelet-flags {
   else # For nodes
     flags+=" ${NODE_KUBELET_TEST_ARGS:-}"
     flags+=" --bootstrap-kubeconfig=/var/lib/kubelet/bootstrap-kubeconfig"
-    flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
+    
+    if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
```

Comment:
this else block is for NODEs, any reason we still need this if-else block here to check the ARKTOS_SCALEOUT_SERVER_TYPE?

Reply:
we only use "ARKTOS_SCALEOUT_SERVER_TYPE" for scale-out, for scale-up, ARKTOS_SCALEOUT_SERVER_TYPE=null, so here need if-else to give different value for scale-out and scale-up.